source/includes/getting-started/_install.html.md

```diff
@@ -0,0 +1,265 @@
+# Install Prerequisites
+
+[//]: # (If any commented statements become false, change prior text)
+
+Before you begin, you will need to have access to our Beta. If you don't have access yet, request access at [bons.ai][1] and then come visit us in the [Forums][17] and let us know why you're excited to try out Bonsai!
+
+The next section walks you through how to install Git, Python, and Pip, but if you're already ahead of the curve and **have them all installed**, skip down to [Setup the Bonsai CLI][2]. 
+
+<aside class="notice">
+We highly recommend Mac users use <a href="http://brew.sh/">Homebrew</a> or download Python directly in the Manual Install instructions so you can perform the following steps on user-land Python rather than <a href="https://github.com/MacPython/wiki/wiki/Which-Python">system Python</a>.
```

Comment:
I would list manual install before Homebrew since we list manual first in the nav.